msal-dotnet-articles/advanced/webview2.md

```diff
@@ -5,44 +5,47 @@ description: "How to use the modern embedded browser based on Microsoft Edge wit
 |  .NET 5         | WebView2, fallback to Legacy* | Embedded |
 
-*_In .NET Core and .NET 5, fallback to legacy WebView is available starting in MSAL 4.30.0._
+*_In .NET Core and .NET 5+, fallback to legacy web view is available starting in MSAL 4.30.0._
 
 ## Troubleshooting
 
 ### WebView2 on .NET Framework
 
-There's a scenario when an app that targets .NET Framework and references MSAL.NET NuGet package tries to acquire token interactively with WebView2 embedded browser, WebView2 will throw exceptions. These errors can be `System.BadImageFormatException: An attempt was made to load a program with an incorrect format.` or `System.DllNotFoundException: 'Unable to load DLL 'WebView2Loader.dll': The specified module could not be found.` (As of MSAL.NET 4.32.0 these exceptions are wrapped into `MsalClientException`.)
+There's a scenario when an app that targets .NET Framework and references the [MSAL for .NET NuGet package](https://www.nuget.org/packages/Microsoft.Identity.Client/) tries to acquire token interactively with WebView2 embedded browser, WebView2 will throw exceptions. These errors can be `System.BadImageFormatException: An attempt was made to load a program with an incorrect format.` or `System.DllNotFoundException: 'Unable to load DLL 'WebView2Loader.dll': The specified module could not be found.` As of MSAL.NET 4.32.0 these exceptions are wrapped into [`MsalClientException`](xref:Microsoft.Identity.Client.MsalClientException).
 
-This occurs because of an existing bug in the WebView2 SDK on .NET Classic platform because it can't figure out the target platform of the dependencies. One possible workaround is to add a `<PlatformTarget>` with values `AnyCPU`, `x86`, or `x64` to your app's project file. (`x86` or `x64` have to match the target framework of the WebView2 installed on the machine.) Another workaround is to add `<PlatformTarget>AnyCPU</PlatformTarget>` in your app's project file and also directly reference WebView2 NuGet. For details, see issues [#2482](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2482), [#730](https://github.com/MicrosoftEdge/WebView2Feedback/issues/730#issuecomment-803132248).
+This occurs due to an existing bug in the WebView2 SDK on the .NET Framework platform because it the library cannot figure out the target platform of its dependencies. One workaround is to add a `<PlatformTarget>` with values `AnyCPU`, `x86`, or `x64` to your application project file. `x86` or `x64` have to match the target framework of the WebView2 installed on the machine. Another workaround is to add `<PlatformTarget>AnyCPU</PlatformTarget>` in your app's project file and also directly reference the [WebView2 NuGet package](https://www.nuget.org/packages/Microsoft.Web.WebView2). For details, see issues [#2482](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2482), [#730](https://github.com/MicrosoftEdge/WebView2Feedback/issues/730#issuecomment-803132248).
```

Comment:
```suggestion
This occurs due to an existing bug in the WebView2 SDK on the .NET Framework platform because the library cannot figure out the target platform of its dependencies. One workaround is to add a `<PlatformTarget>` with values `AnyCPU`, `x86`, or `x64` to your application project file. `x86` or `x64` have to match the target framework of the WebView2 installed on the machine. Another workaround is to add `<PlatformTarget>AnyCPU</PlatformTarget>` in your app's project file and also directly reference the [WebView2 NuGet package](https://www.nuget.org/packages/Microsoft.Web.WebView2). For details, see issues [#2482](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2482), [#730](https://github.com/MicrosoftEdge/WebView2Feedback/issues/730#issuecomment-803132248).
```